tests/test_dvc.py

```diff
@@ -60,31 +57,6 @@ def test_exp_save_skip_on_env_vars(tmp_dir, monkeypatch, mocker):
     assert live._inside_dvc_pipeline
 
 
-def test_exp_save_run_on_dvc_repro(tmp_dir, mocker):
```

Comment:
This test was actually outdated and not working. It conflicts with `test_dvc_repro` below, so I consolidated the meaningful assertions into that test.